.github/workflows/ci.yml

```
@@ -101,12 +101,12 @@ jobs:
           platforms: linux/amd64
           build-args: ${{ matrix.build-args }}
           context: ${{ matrix.cache_scope }}/
```

I think we can clean up this var name now too:
```suggestion
          context: ${{ matrix.context }}/
```

---

Can't see `matrix.context` being defined for any of the entries? Do you mean `image_name` perhaps?

---

`cache_scope` -> `context` in the matrix definition here:
https://github.com/firezone/firezone/pull/2246/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fL37

---

`s/cache_scope/context/` probably suffices 🎯